DS and Algorithm/SeleniumTest/SeleniumExample.java

```diff
@@ -0,0 +1,43 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
+
+        // Navigate to the desired website
+        driver.get("https://www.google.com");
+
+        // Perform a search using WebInteraction class
+        WebInteraction.performSearch(driver, "Selenium WebDriver");
+
+        // Wait for a few seconds to see the search results
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        // Click on the first search result
+        WebInteraction.clickOnResult(driver, 1);
+
+        // Wait for a few seconds to see the result page
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+
+        // Create a SearchResultsPage instance and get the number of results
+        SearchResultsPage resultsPage = new SearchResultsPage(driver);
+        int numberOfResults = resultsPage.getNumberOfResults();
+        System.out.println("Number of search results: " + numberOfResults);
+
+         // Create a SearchResultsPage instance and get the number of results
+         SearchResultsPage resultsPage = new SearchResultsPage(driver);
+         int numberOfResults = resultsPage.getNumberOfResults();
+         System.out.println("Number of search results: " + numberOfResults);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/11#issuecomment-2160369457>#04e21a</a><span> - 06/11/2024, 10:17 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Duplicate code block: The code block that creates a 'SearchResultsPage' instance and prints the number of results is duplicated.
</div>
<div id="fix">
<b>Fix:</b> Remove the duplicate code block to avoid redundancy and potential confusion.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -34,8 +34,6 @@
  // Create a SearchResultsPage instance and get the number of results
  SearchResultsPage resultsPage = new SearchResultsPage(driver);
  int numberOfResults = resultsPage.getNumberOfResults();
  System.out.println("Number of search results: " + numberOfResults);

 - // Create a SearchResultsPage instance and get the number of results
 - SearchResultsPage resultsPage = new SearchResultsPage(driver);
 - int numberOfResults = resultsPage.getNumberOfResults();
 - System.out.println("Number of search results: " + numberOfResults);
</code></pre>
</div>
</details>
</div>
<br>


</div>